.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -46,6 +46,7 @@ Steps to reproduce the behavior:
 ### Debuglog
 <!--- Put your text below this line -->
 <!--- A debuglog is always mandatory when creating an issue. Provide one! -->
+<!--- Use https://paste.kodi.tv -->
```

Comment:
i suggest also to add
`or attach the log file by adding the ".txt" extension`
because with kodi paste you cannot upload logs with big sizes where instead GH allow it

Reply:
I correct myself github now allow also .log extension so there is no problem to rename extension

Reply:
Okay, you can make suggestion via GitHub next time.

```suggestion
<!--- Use https://paste.kodi.tv or upload log file as attachment to the issue -->
```

Reply:
good